resources/neumann-testnet-parachain.yaml

```diff
@@ -0,0 +1,67 @@
+networkSpec:
+    displayName: neumann-parachain-testnet-jzia5
+    name: neumann-parachain-testnet-jzia5
+    protocol: polkadot-parachain
+    imageRepository: oaknetwork/neumann
+    imageVersion: 1.2.4-rc2
+    config:
+      nodeTypes:
+        full:
+          args:
+            - key: "--force-authoring"
+            - key: --chain
+              file: "neumann.json"
+            - key: --parachain-id
+              value: 2000
+            - key: --
+            - key: --execution
+              value: wasm
+            - key: --chain
+              file: "rococo-testnet.json"
+            - key: --bootnodes # Relay chain p2p address
+              value: /dns4/node-6896992096614604800-0.p2p.onfinality.io/tcp/23160/ws/p2p/12D3KooWM1vTue78y2ZjxTjZKgYEPR8gSNFtTda3KETgp3WAhCxK
+        collator:
+          args:
+            - key: "--force-authoring"
+            - key: --chain
+              file: "neumann.json"
+            - key: --parachain-id
+              value: 2000
+            - key: --collator
+            - key: --bootnodes # Relay chain p2p address
+              value: FILL_ME_IN
+            - key: --
+            - key: --execution
+              value: wasm
+            - key: --chain
+              file: "rococo-testnet.json"
+            - key: --bootnodes # Relay chain p2p address
+              value: /dns4/node-6896992096614604800-0.p2p.onfinality.io/tcp/23160/ws/p2p/12D3KooWM1vTue78y2ZjxTjZKgYEPR8gSNFtTda3KETgp3WAhCxK
+validator:
+  count: 2
+  node:
+    nodeName: neumann-parachain-testnet-collator-jzia5
+    nodeType: collator
```

Comment:
does this work bnow?

Reply:
it doesn't by itself cause of hte bootnode. you need to manually create the bootnode and connect the P2P address and remove the validator param. after that, it works. but it saves like 10 min each time from creating the network and collators manually haha

Reply:
can we add a comment somewhere about hpw this is just to save time but won't work without UI edits

Reply:
for sure - lemme add! also will def add in the doc to run the neumann testnet